exchanges/bitfinex/bitfinex_test.go

```diff
@@ -1258,67 +1260,58 @@ func TestSubToMap(t *testing.T) {
 	assert.Equal(t, "tBTCLTC", r["symbol"], "symbol should not use colon delimiter if both currencies < 3 chars")
 }
 
-// TestWsPlaceOrder dials websocket, sends order request.
-func TestWsPlaceOrder(t *testing.T) {
+// TestWSNewOrder exercises WsNewOrder
```

Comment:
```suggestion
// WSNewOrder exercises WSNewOrder
```
For this one and the others, though we can rid these comments as they're self explanatory and the linter only complains on non-exported test code comments

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/92686e35d